Added patrol outcomes & tagged pronouns/verbs

Adamented opened the pull request:
fst_hunt_newleaf_ratswarm1 got 2 more success outcomes and is tagged with pronouns and conjugate verbs.

Adamented committed:
Added patrol outcomes & tagged pronouns/verbs

fst_hunt_newleaf_ratswarm1 got 2 more success outcomes and is tagged with pronouns and conjugate verbs.

keyraven commented:
Looks good! 
In the future, I suggest making your edits on a fork, and PR from that fork. It makes things much simpler, and in generally good practices. 
If you already have an existing fork for modding purposes, you can create new branches specifically for editing and PR, so it doesn't effect your existing branches.

Adamented committed:
Update newleaf.json

fst_hunt_newleaf_ratswarm1 fixed a group "they" tagged as pronoun, fixed rc_has_stat tag and changed in code to s_c (r_c with skill will be the same as s_c)
